samples/SwiftDemoApp/SwiftDemoApp/KMLViewController.swift

```diff
@@ -27,17 +27,15 @@ class KMLViewController: UIViewController {
     let camera = GMSCameraPosition.camera(withLatitude: 37.4220, longitude: -122.0841, zoom: 17)
     mapView = GMSMapView.map(withFrame: CGRect.zero, camera: camera)
     self.view = mapView
-
-    let path = Bundle.main.path(forResource: "KML_Sample", ofType: "kml")
-    let url = URL(fileURLWithPath: path!)
```

Comment:
I would rather have this error if for some reason the resource is not available vs. failing silently without any notice. Can you add a log statement for the error case?

Reply:
Added `NSLog`